app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -176,25 +176,29 @@ def parse_openshift_image(openshift_image)
         ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id
```

Comment:
why not the following:
```ruby
id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
ref = "ContainerImage::DOCKER_PULLABLE_PREFIX#{id}"
```

or 
```ruby
id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id.to_s
```
In other words, make creation of the `ref` defensive?